Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -1831,19 +2072,333 @@ def export_pcap_file_command(client: Client, args: Dict) -> List:
+    """
+    device_id = arg_to_number(args.get('device_id'))
+    domain_id = arg_to_number(args.get('domain_id'))
+    limit = arg_to_number(args.get('limit', 50))
+    all_results = argToBoolean(args.get('all_results', False))
+
+    response = client.list_device_interface_request(domain_id=domain_id, device_id=device_id)
+
+    interfaces = (response.get('allocatedInterfaceList', []) if all_results
+                  else
+                  response.get('allocatedInterfaceList', [])[:limit])
+
+    key_list = ['interfaceId', 'interfaceName', 'interfaceType']
+    # Capitalize the keys of the interfaces list and keeping the rest of the keys as is.
+    capitalize_interfaces = []
+    for interface in interfaces:
+        interface = {k[:1].upper() + k[1:]: v for k, v in interface.items() if k in key_list}
+        capitalize_interfaces.append(interface)
+
+    readable_output = tableToMarkdown(
+        name='Device interfaces List', t=capitalize_interfaces, removeNull=True
+    )
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='NSM.Interface',
+        outputs=capitalize_interfaces,
+        raw_response=response
+    )
+
+
+def assign_device_policy_command(client: Client, args: Dict) -> CommandResults:
+    """
+    Args:
+        client(Client): client - A McAfeeNSM client.
+        args(Dict): - The function arguments.
+    Returns: A CommandResult object with a success or failure message.
+    """
+    device_id = arg_to_number(args.get('device_id'))
+    domain_id = arg_to_number(args.get('domain_id'))
+    pre_firewall_policy = args.get('pre_firewall_policy_name')
+    post_firewall_policy = args.get('post_firewall_policy_name')
+
+    response = client.assign_device_policy_request(domain_id=domain_id, device_id=device_id,
+                                                   pre_firewall_policy=pre_firewall_policy,
+                                                   post_firewall_policy=post_firewall_policy
+                                                   )
+
+    return CommandResults(
+        readable_output='Policy assigned successfully.',
+        raw_response=response
+    )
+
+
+def list_device_policy_command(client: Client, args: Dict) -> CommandResults:
+    """
+
+    Args:
+        client(Client): client - A McAfeeNSM client.
+        args(Dict): - The function arguments.
+
+    Returns:
+        CommandResults: A CommandResult object with a list of device policies.
+    """
+    device_id = arg_to_number(args.get('device_id'))
+    domain_id = arg_to_number(args.get('domain_id'))
+    limit = arg_to_number(args.get('limit', 50))
+    all_results = argToBoolean(args.get('all_results', False))
+    response = client.list_device_policy_request(domain_id=domain_id, device_id=device_id)
+
+    all_policies = (response.get('policyAssignmentsList') if all_results
+                    else
+                    response.get('policyAssignmentsList', {})[:limit])
```

Comment:
I think it should be a list - please verify that. 
```suggestion
                    response.get('policyAssignmentsList', [])[:limit])
```